packages/botonic-plugin-hubtype-analytics/src/types.ts

```diff
@@ -0,0 +1,150 @@
+import { PROVIDER } from '@botonic/core'
+
+export interface Event {
+  chat: string
+  event_type: EventName
+  event_data: BaseEventData
+}
+
+export enum EventName {
+  botAgentRating = 'bot_agent_rating',
```

Comment:
don't we usually use upper case for enum items?

Reply:
I have always used PascalCase for enum and camelCase for enum items